readme.md

```diff
@@ -1,139 +1,47 @@
-From here, synchronize with the blockchain; this takes around 2-5 hours on the testnet and up to a day using Bitcoin core.
-</li>
-<li>
-Install Go in your pc and get it working on your compiler/IDE. The guide below will refer to installing Go on VSCode.
-</li>
-<li>
-
-Get the Utreexo Code from ```github.com/mit-dci/utreexo```
-</li>
-<li>
-
-Build and generate proofs (server) by running the following command where $USER is your username
-```
-go build bridgeserver.go
-bridgeserver -datadir=C:\Users\$USER\AppData\Roaming\Bitcoin\testnet3\blocks\
-```
- **If this fails, the command run was interupted or failed. To relaunch, delete the folders in the Utreexo\utreexoserver\utree folder: forestdata, offsetdata, pollarddata, proofdata, testnet-ttlbd**
-
-</li>
-<li>
-
- For a debugging session, in launch.json in VSCode, create a configuration as follows. The
-      ```"-datadir"```    argument should point to the folder where Bitcoin was downloaded in step 1.
-
-```
-{
-    "version": "0.2.0",
-    "configurations": [
-        {
-            "name": "Launch",
-            "type": "go",
-            "request": "launch",
-            "mode": "auto",
-            "program": "${fileDirname}",
-            "env": {},
-            "args": ["-datadir=C:\\Users\\$USER\\AppData\\Roaming\\Bitcoin\\testnet3\\blocks\\"]
-        }
-    ]
-}
-```
-</li>
-
-<li>
-
-Finally run Utreexo client from  **command line** using the following. Make sure that the server has finished running before running this command.
-
-```
-go build csnclient.go
-C:\utreexo\csnclient -datadir=C:\Users\admin\AppData\Roaming\Bitcoin\testnet3\blocks\
-```
-</li>
-</ol>
+Utreexo pushes the costs of maintaining the network to the right place: an exchange creating millions of transactions may need to maintain millions of proofs, while a personal account with only a few unspent outputs will only need to maintain a few kilobytes of proof data. Utreexo also provides a long-term scalability solution as the accumulator size grows very slowly with the increasing size of the underlying set (the accumulator size is logarithmic with the set size)
+
+## Development Process
+
+At the moment, this repository holds the code for the accumulator and proof-of-concept implementations for the compact state node and the bridge node. The accumulator package is currently used in [mit-dci/utcd](https://github.com/mit-dci/utcd) in an alternative full node bitcoin implementation written in Go.
+
+The c++ implementation of UTreeXO is at [mit-dci/libutreexo](https://github.com/mit-dci/libutreexo) and the package is currently being used in [dergoegge/bitcoin](https://github.com/dergoegge/bitcoin) in an full node bitcoin-core implementation.
```

Comment:
arg at UTreeXO ಠ_ಠ.

It doesn't really matter but it hurts me.

Reply:
Should I change it to `utreexo` instead of `UTreeXO`?

I thought the capitalization was standard `:')`

Reply:
I'd prefer it to be changed to utreexo. Idk some random people write it as UTreeXO but Tadge, Niklas and I don't call it that so...